### PR TITLE
add orcid springer https://github.com/ubtue/DatenProbleme/issues/1526

### PR DESCRIPTION
**Pull request by socheres**

### CODE REVIEW

File: Springer Link.js [hunk 1]
```diff
@@ -225,8 +226,23 @@ function complementItem(doc, item) {
...
+	getORCID(doc, item);
 	return item;
 }
+	// ORCID
+function getORCID(doc, item) {
+	let authorOrcidEntries = ZU.xpath(doc, '//*[@class="c-article-author-list__item"]');
+	for (let authorOrcidEntry of authorOrcidEntries) {
+		let authorEntry = authorOrcidEntry.innerText;
+		let orcidEntry = authorOrcidEntry.innerHTML;
+		if (authorEntry && orcidEntry.match(/\d+-\d+-\d+-\d+x?/i)) {
```
Comment (jriedl) [inline Springer Link.js:239]: Please test `orcidEntry` for undefined before executing `match`.

Comment (socheres) [reply Springer Link.js:239]: Do you mean like `if (authorEntry && typeof orcidEntry !== 'undefined' && orcidEntry.match(/\d+-\d+-\d+-\d+x?/i))` or should be enough like this? `if (authorEntry && orcidEntry && orcidEntry.match(/\d+-\d+-\d+-\d+x?/i))`

Comment (jriedl) [reply Springer Link.js:239]: The second one should be sufficient.

**Comment by jriedl** [comment]
Seit kurz nach dem Merge auf nu.

**Pull request merged**